SQL/beestation_schema.sql

```diff
@@ -82,36 +82,40 @@ CREATE TABLE IF NOT EXISTS `SS13_ban` (
 -- Dumping structure for table ss13tgdb.SS13_characters
 DROP TABLE IF EXISTS `SS13_characters`;
 CREATE TABLE IF NOT EXISTS `SS13_characters` (
-  `real_name` varchar(50) DEFAULT NULL,
-  `name_is_always_random` bit(1) NOT NULL,
-  `body_is_always_random` bit(1) NOT NULL,
-  `gender` varchar(50) NOT NULL,
-  `age` tinyint(3) unsigned NOT NULL,
-  `hair_color` varchar(50) NOT NULL,
-  `facial_hair_color` varchar(50) NOT NULL,
-  `eye_color` varchar(50) NOT NULL,
-  `skin_tone` varchar(50) NOT NULL,
-  `hair_style_name` varchar(50) NOT NULL,
-  `facial_style_name` varchar(50) NOT NULL,
-  `feature_ethcolor` varchar(50) NOT NULL,
-  `underwear` varchar(50) NOT NULL,
-  `undershirt` varchar(50) NOT NULL,
-  `socks` varchar(50) NOT NULL,
-  `backbag` varchar(50) NOT NULL,
-  `uplink_loc` varchar(50) NOT NULL,
-  `species` varchar(50) NOT NULL,
-  `features` json NOT NULL,
-  `joblessrole` smallint(6) NOT NULL,
-  `job_civlian_high` smallint(6) NOT NULL,
-  `job_civilian_med` smallint(6) NOT NULL,
-  `job_civilian_low` smallint(6) NOT NULL,
-  `job_medsci_high` smallint(6) NOT NULL,
-  `job_medsci_med` smallint(6) NOT NULL,
-  `job_medsci_low` smallint(6) NOT NULL,
-  `job_engsec_high` smallint(6) NOT NULL,
-  `job_engsec_med` smallint(6) NOT NULL,
-  `job_engsec_low` smallint(6) NOT NULL
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+	`slot` INT(11) UNSIGNED NOT NULL,
+	`ckey` VARCHAR(64) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`species` VARCHAR(32) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`real_name` VARCHAR(64) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`name_is_always_random` BIT(1) NOT NULL,
+	`body_is_always_random` BIT(1) NOT NULL,
```

Comment:
```suggestion
	`name_is_always_random` tinyint(1) unsigned NOT NULL,
	`body_is_always_random` tinyint(1) unsigned NOT NULL,
```

Missed this on my initial review, booleans should be stored as tinyints, not as bit characters. No code changes should be needed with this change.

Reply:
https://mariadb.com/kb/en/boolean/ for more info